Commands/ImportPaymentMethodsCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
```

Comment:
Maybe in a different PR we add Copyright to these new files

Reply:
@acampos1916 

Do you want us to add the copy right to all new files?
If so could you provide us with the desired text to be placed in the files.

Reply:
@bortefi indeed we would like to have [this](https://github.com/Adyen/adyen-shopware6/blob/develop/src/AdyenPaymentShopware6.php#L2-L23) text in all the files, but I noticed that there is no copyright info in any of the files in the repo so we can also just add that in a separate PR